### ISSUE TITLE
Crash on Android 8.0.0 using release 4.1.0

### ISSUE THREAD

**Issue by masterwok** [spacecowboy/NoNonsense-FilePicker#165]
Hi, I'm seeing the following crash using release 4.1.0 on Android 8.0.0 devices:

```
Caused by java.lang.IllegalStateException: Only fullscreen opaque activities can request orientation
       at android.app.Activity.onCreate(Activity.java:986)
       at android.support.v4.app.SupportActivity.onCreate(SupportActivity.java:66)
       at android.support.v4.app.FragmentActivity.onCreate(FragmentActivity.java:321)
       at android.support.v7.app.AppCompatActivity.onCreate(AppCompatActivity.java:84)
       at com.nononsenseapps.filepicker.AbstractFilePickerActivity.onCreate(AbstractFilePickerActivity.java:84)
       at android.app.Activity.performCreate(Activity.java:6975)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1214)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2775)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2897)
       at android.app.ActivityThread.-wrap11(Unknown Source)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1598)
       at android.os.Handler.dispatchMessage(Handler.java:105)
       at android.os.Looper.loop(Looper.java:251)
       at android.app.ActivityThread.main(ActivityThread.java:6572)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
```

**Comment by masterwok** [comment]
```
    compileSdkVersion 27
    defaultConfig {
        minSdkVersion 16
        targetSdkVersion 27
    }
```

**Comment by masterwok** [comment]
I believe this has to do with setting screen orientation to portrait.

**Comment by masterwok** [comment]
This only occurs on API 26

**Comment by spacecowboy** [comment]
Did you find a solution to the problem?

**Comment by masterwok** [comment]
Hi, sorry for the delayed response. I initially had orientation set to portrait in my manifest for the file picker activity, but this was causing the crash. I also tried pushing an update to my app where I set the requested orientation programmatically, but it appears the crash still occurred for the user. Unfortunately, I was never able to reproduce this issue on my device. My "solution" was to just allows orientation changes by removing the request for portrait orientation. To me, this is more of a work around than a fix but it is not a critical issue for my app.
